<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSerialPortTestClass</class>
 <widget class="QMainWindow" name="QtSerialPortTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtSerialPortTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir le port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="portStatusLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status port : Fermé</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Données reçues :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portComboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="receivedDataText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1369</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtSerialPortTest.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtSerialPortTestClass</receiver>
   <slot>onOpenPortButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtSerialPortTestClass</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenPortButtonClicked()</slot>
  <slot>onSerialPortReadyRead()</slot>
  <slot>onSendMessageButtonClicked()</slot>
 </slots>
</ui>
